include/vara/Feature/Constraint.h

```diff
@@ -570,10 +570,10 @@ class PrimaryConstraint : public Constraint {
 
 class PrimaryIntegerConstraint : public PrimaryConstraint {
 public:
-  PrimaryIntegerConstraint(int Value)
+  PrimaryIntegerConstraint(long Value)
```

Comment:
```suggestion
  PrimaryIntegerConstraint(int64_t Value)
```

I would use `int64_t` instead
a) it makes the size clear
b) it also works for 32bit latforms